.buildkite/common.py

```diff
@@ -89,11 +90,18 @@ def pipeline_to_json(pipeline):
     return json.dumps(pipeline, indent=4, sort_keys=True, ensure_ascii=False)
 
 
-def get_changed_files(branch):
+def get_changed_files():
     """
     Get all files changed since `branch`
     """
-    stdout = subprocess.check_output(["git", "diff", "--name-only", branch])
+    # Files are changed only in context of a PR
+    if os.environ.get("BUILDKITE_PULL_REQUEST", "false") == "false":
+        return []
+
+    branch = os.environ.get("BUILDKITE_PULL_REQUEST_BASE_BRANCH", "main")
+
+    stdout = subprocess.check_output(f"git diff --name-only origin/{branch}".split(" "))
```

Comment:
nit: may be worth doing it in list form for clarity.

Reply:
sure, will remember the next time! but since the current is just a backport I would like to go ahead since this is just a nit.